tensordata/kaplonekVaccine2022/Luminex-functional-assay.csv

```diff
@@ -17,7 +17,7 @@ Male,32.57573165,BNT162b2,320.5,123.1,150.94899,114.95,109.08,1.100135,0.58529,1
 Female,27.86641752,mRNA-1273,421,119.98,143.23972,132.18,117.34,1.1048565,0.334698,0.78424825,0.495159,32.15,6.875,48.35,2650,782,184,1600,2479,1319.5,1575.5,1392.5,49.8,25.4,25.4,49.8,98.7,49.8,233,25.4,2186,550.5,172,1429,4347,818.5,1050.5,745.5,1075.5,404,86.45,562.5,1929.5,1038.5,1051,892,416,282,184,465,709,465,440,428,18982.5,9644.5,2015.5,14722.5,17566.5,13672.5,21949,16651,19776,4236.5,135,12403.5,18384.5,10267,21167.5,15967.5,17701,10120.5,2503.5,13306.5,16309.5,12671.5,18555.5,14905.5,18604.5,8009,379.5,12855,16712.5,11866,16676,13453
 Female,23.22851713,mRNA-1273,285.5,100.98,131.792055,124.69,97.1,1.180055,0.52304,1.0294475,0.98763,33.75,8.455,56.4,4212.5,2137,221,3492,1978.5,2625,2149.5,2454.5,49.8,25.4,25.4,49.8,98.7,49.8,74.2,49.8,1612,514,270,1002,6349,929,1173,1026,416,122.85,62,171.5,2125,379,477.5,379,147,74.2,49.8,98.7,685,221,196,245,24414.5,16737,4005,20472,18323.5,19556.5,22144.5,20447.5,23035,13233,257.5,19434.5,14905.5,17750,22254.5,20277,17164,11598,3126,13429,12599,12159,15626,13575,22022,13160,1039,17152,15747.5,16724.5,16956.5,18299
 Male,22.52487503,mRNA-1273,351.5,86.72,112.829345,100.92,87.68,0.89062,0.51952,0.95062,1.16694,19.91,4.69,43.2,37.6,25.4,25.4,49.8,74.25,74.2,233,110.85,575,172,49.8,526,245,306,989.5,452.5,2247,623.5,245,1514.5,2894,904.5,989.5,648,1271,440,98.7,758,1197,1026,2076,1466,74.2,74.2,172,233,196,147,587,306,14063,5274.5,342.5,10083.5,13148,9022,17823.5,10340.5,12122.5,1026.5,62,5872.5,12867,3748.5,17798.5,7313,13184.5,6275.5,233,9998.5,12439.5,9180.5,15674.5,9339,13526,2564.5,86.45,8094.5,12635,6263,12757,5970
-Male,34.56619004,mRNA-1273,433.5,95.26,138.91746,114.25,87.7,1.91844,0.9065,1.663825,1.38003,39.75,8.625,55.35,4798.5,1661,245.5,3309,4395.5,2027.5,2247,1722,49.8,25.4,25.4,49.8,49.8,74.2,135,49.8,2748,1234,342.5,1954.5,3675.5,2564.5,2845,2418,330.5,184,98.7,221,416,489,941,819,294,245,98.7,270,306,379.5,465,416,30323,18494.5,6349,23878,27882,22120,25525.5,22510.5,26526.5,15027.5,501.5,22144.5,24073,19788.5,25892,22437,25354.5,16431.5,6458.5,20289,24219.5,18409,20850.5,18128.5,25318.5,16139,2259.5,20386.5,23792,19495.5,20582,18360
+Male,34.56619003,mRNA-1273,433.5,95.26,138.91746,114.25,87.7,1.91844,0.9065,1.663825,1.38003,39.75,8.625,55.35,4798.5,1661,245.5,3309,4395.5,2027.5,2247,1722,49.8,25.4,25.4,49.8,49.8,74.2,135,49.8,2748,1234,342.5,1954.5,3675.5,2564.5,2845,2418,330.5,184,98.7,221,416,489,941,819,294,245,98.7,270,306,379.5,465,416,30323,18494.5,6349,23878,27882,22120,25525.5,22510.5,26526.5,15027.5,501.5,22144.5,24073,19788.5,25892,22437,25354.5,16431.5,6458.5,20289,24219.5,18409,20850.5,18128.5,25318.5,16139,2259.5,20386.5,23792,19495.5,20582,18360

```

Comment:
How did this change?

Reply:
There’s no sample number in this so we use age as identifier. Two people have identical age so I made a small change to make it unique.